docs/source/newsfragments/3990.feature.rst

```diff
@@ -0,0 +1 @@
+Altair DSim simulator is now supported by cocotb.
```

Comment:
```suggestion
The Siemens DSim simulator is now supported by cocotb.
```
(not sure if the sale is completed yet)

Reply:
I guess it is: https://press.siemens.com/global/en/pressrelease/siemens-acquires-altair-create-most-complete-ai-powered-portfolio-industrial-software

I changed Altair to Siemens but the links to install Dsim are still on Altair Marketplace.